lib/api/BackbeatServer.js

```diff
@@ -203,9 +205,40 @@ function run(config, Logger) {
     const logger = new Logger('BackbeatServer');
     const apiLogger = new Logger('BackbeatAPI');
     const cluster = new Clustering(WORKERS, logger);
-    const backbeatAPI = new BackbeatAPI(config, apiLogger);
+    let backbeatAPI;
 
-    backbeatAPI.setupInternals(err => {
```

Comment:
This was done before the workers were started (done below by `cluster.start()`), so I don't think there were multiple producers/consumers created in the previous code?

Reply:
And, even if we spawn a consumer per worker, Kafka advertises that maintaining a consumer is very cheap on server side, I don't know about the consumer client but it should be relatively low footprint for what it has to do, not sure this overhead is a real concern.

As discussed offline, I don't know if there is a need to have workers at all, unless there are CPU intensive tasks -
 @rahulreddy should we keep the cluster workers in your opinion?

Reply:
Ah sorry I see now. For now, I will close this PR and revise the original healthcheck PR for checking all current BB topics 👍 

For the workers, I talked with @rahulreddy offline and he suggested we keep it to 2 workers for now in the case where something should happen to one of the workers. Switching to a single http server should be fairly quick if needed as well